.python-version

```diff
@@ -0,0 +1 @@
+3.9
```

Comment:
Using 3.9 as 3.10 and 3.11 both have issues compiling the `[ml]` options in pip install.  Opening tickets to address this.